src/sentry/api/helpers/group_index/delete.py

```diff
@@ -53,12 +54,28 @@ def delete_group_list(
         if g.issue_category == GroupCategory.ERROR:
             error_ids.append(g.id)
 
+    transaction_id = uuid4().hex
+    delete_logger.info(
+        "object.delete.api",
+        extra={
+            "objects": group_ids,
+            "project_id": project.id,
+            "transaction_id": transaction_id,
+        },
+    )
+    # The tags can be used if we want to find errors for when a task fails
+    sentry_sdk.set_tags(
+        {
+            "project_id": project.id,
+            "transaction_id": transaction_id,
+        },
+    )
```

Comment:
Should we use an isolation scope for these tags?

Reply:
The docs seem to indicate we can use the top-level call:
https://docs.sentry.io/platforms/python/enriching-events/scopes/#changing-the-scope
> We generally recommend using the top-level API to manage your scopes, since the SDK's automatic scope management handles most use cases.

I have also noticed that the Sentry app does not leak tags anymore, thus, I assume the SDK folks fixed the issue.

I'm asking on #discuss-backend

Reply:
That's great news. Tags leaking across requests was my concern.